CharactersPage.js

```diff
@@ -655,6 +649,13 @@ function observe_character_theme_change() {
             // console.log("theme_observer is calling find_and_set_player_color", mutation, node);
             const newColor = node.innerHTML.match(/#(?:[0-9a-fA-F]{3}){1,2}/)?.[0];
             if (newColor) {
+              let button = $("#avtt-character-join-button");
+              //$(".ct-character-sheet-desktop > .ct-character-header-desktop > .ct-character-header-desktop__group--gap");
+              let color = $(".ct-character-header-desktop__button").css("border-color") ?? "black";
```

Comment:
You can find the theme color a little easier in all layouts with $('.dice-toolbar').css('--dice-color') 
Currently this doesn't look like it grabs it in mobile layout.

I find it a little distracting being fully colored rather than just the border like the other buttons. I'm sure I'd get used to it but if you guys like it though I'm good with putting it in I use my own theme anyway lol.